components/patient-data/api/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1,16 @@
+phenotips.patientData.buttons.cancel=Cancel
+phenotips.patientData.buttons.import=Import
+phenotips.patientData.buttons.importFileSelect=select a local file
+
+phenotips.patientData.jsonImportDialog.longWaitWarning=Importing may take a while, please be patient.
+phenotips.patientData.jsonImportDialog.importSuccess=Patients were successfully imported.
+phenotips.patientData.jsonImportDialog.notAuthenticated=Failed: User authentication failed.
+phenotips.patientData.jsonImportDialog.notAuthorized=Failed: User not authorized.
+phenotips.patientData.jsonImportDialog.patientCreateFailed=Failed: Could not import patient data.
+
+phenotips.patientData.jsonImportDialog.fileUploadNotSupported=Your browser does not support file upload. Please copy-paste import data in the box above.
+
+phenotips.patientData.jsonImportDialog.singleImport.title=Import patient from JSON
+phenotips.patientData.jsonImportDialog.multiImport.title=Import patients from JSON
+phenotips.patientData.jsonImportDialog.sectionTitles.import=Paste a JSON in the box below, or
```

Comment:
I wonder if this will work in all languages or if it's better to put the "Select a local file" button as a parameter. I think a parameter is safer based on what Google translate produced for Japanese (not that I understand any of it unfortunately).